dump_zenodo_open_metadata.py

```diff
@@ -53,53 +53,68 @@ def parse_record(rec, spam=False):
     out["spam"] = spam
     return out
 
+DUMP_TYPES = {
+  '1': 'ham',
+  '2': 'spam',
+  '3': 'full',
+}
+
+choices = [k + ' - ' + v for k, v in DUMP_TYPES.items()]
+choice = str(input("What type of dump you want? {}".format(', '.join(choices)) + '\n'))
```

Comment:
```suggestion
choice = input("What type of dump you want? {}".format(', '.join(choices)) + '\n')
```

I think what comes back from `input` is always a string (that's why I put the keys in `DUMP_TYPES` as strings :) )